lighthouse-core/gather/gatherers/form-elements.js

```diff
@@ -60,15 +61,17 @@ function collectFormElements() {
           attribute: child.getAttribute('autocomplete'),
           prediction: child.getAttribute('autofill-prediction'),
         },
-        // @ts-expect-error - getNodeDetails put into scope via stringification
-        ...getNodeDetails(child),
+        nodeLabel: nodeDetails.nodeLabel,
+        snippet: nodeDetails.snippet,
       });
     }
     if (child instanceof HTMLLabelElement) {
+      // @ts-expect-error - getNodeDetails put into scope via stringification
+      const nodeDetails = getNodeDetails(child);
       formObj.labels.push({
         for: child.htmlFor,
-        // @ts-expect-error - getNodeDetails put into scope via stringification
-        ...getNodeDetails(child),
+        nodeLabel: nodeDetails.nodeLabel,
```

Comment:
same here.

these'd probably need a change over in the autocomplete audit, but it should be fairly rote.